src/Utilities/Utils.res

```diff
@@ -799,11 +799,15 @@ let delay = timeOut => {
   })
 }
 let getHeaders = (~uri=?, ~token=?, ~headers=Dict.make(), ()) => {
+  let arrayOfNameAndVersion = String.split(Window.userAgent->OrcaLogger.browserDetect, "-")
```

Comment:
We already have this function in orca-logger. Could you please use the same one

Reply:
Using that only.